### PR TITLE
[Template] No code in __init__.py

### PR DESCRIPTION
**Pull request by ayushkovskiy**
also closes https://github.com/neuromation/cookiecutter-neuro-project/issues/265

### CODE REVIEW

**Comment by AlekseySh** [comment]
agree, don't forget about call in start notebook

**Comment by ayushkovskiy** [comment]
> but let's rename 'Untitled' to 'demo'?

Out of scope of current PR, please create an issue. Since https://github.com/neuromation/cookiecutter-neuro-project/pull/207 is merged, `Untitled.ipynb` is not necessary anymore since the user's code will still be available from the notebook. I will drop this notebook at all.

**Comment by ayushkovskiy** [comment]
changed `hello_world.py` -> `train.py` + modified `Makefile`, please re-review

File: {{cookiecutter.project_slug}}/Makefile [hunk 1]
```diff
@@ -43,8 +43,8 @@ PRESET?=gpu-small
 HTTP_AUTH?=--http-auth
 
 # Command to run training inside the environment. Example:
-# TRAINING_COMMAND="bash -c 'cd $(PROJECT_PATH_ENV) && python -u $(CODE_DIR)/train.py --data $(DATA_DIR)'"
-TRAINING_COMMAND?='echo "Replace this placeholder with a training script execution"'
+TRAINING_COMMAND="bash -c 'cd $(PROJECT_PATH_ENV) && python -u $(CODE_DIR)/train.py --data $(DATA_DIR)'"
```
Comment (ayushkovskiy) [inline {{cookiecutter.project_slug}}/Makefile]: how about we change to `TRAINING_COMMAND=python -u $(CODE_DIR)/train.py --data $(DATA_DIR)` and in the target, run:
```
train: _check_setup upload-code upload-config   ### Run a training job
	$(NEURO) run \
		--name $(TRAINING_JOB) \
		--preset $(PRESET) \
		--volume $(DATA_DIR_STORAGE):$(PROJECT_PATH_ENV)/$(DATA_DIR):ro \
		--volume $(PROJECT_PATH_STORAGE)/$(CODE_DIR):$(PROJECT_PATH_ENV)/$(CODE_DIR):ro \
		--volume $(PROJECT_PATH_STORAGE)/$(CONFIG_DIR):$(PROJECT_PATH_ENV)/$(CONFIG_DIR):ro \
		--volume $(PROJECT_PATH_STORAGE)/$(RESULTS_DIR):$(PROJECT_PATH_ENV)/$(RESULTS_DIR):rw \
		${OPTION_GCP_CREDENTIALS} \
		${OPTION_WANDB_CREDENTIALS} \
		--env PYTHONPATH=$(PROJECT_PATH_ENV) \
		--env EXPOSE_SSH=yes \
		--env JOB_TIMEOUT=0 \
		$(CUSTOM_ENV_NAME) \
		"bash -c 'cd $(PROJECT_PATH_ENV) && $(TRAINING_COMMAND)'"
```

File: tests/e2e/test_e2e_template.py [hunk 2]
```diff
@@ -255,11 +255,11 @@ def _run_import_code_in_notebooks_test() -> None:
     )
     job_id = parse_job_id(out)
 
-    expected_string = "----\r\nHello World!\r\n----"
+    expected_string = "----\r\nYour training script here\r\n----"
 
     out_file = f"/tmp/out-nbconvert-{MK_PROJECT_SLUG}"
     jupyter_nbconvert_cmd = "jupyter nbconvert --execute --no-prompt --no-input"
-    notebook_path = f"{MK_PROJECT_PATH_ENV}/{MK_NOTEBOOKS_DIR}/Untitled.ipynb"
+    notebook_path = f"{MK_PROJECT_PATH_ENV}/{MK_NOTEBOOKS_DIR}/hello_world.ipynb"
```
Comment (ayushkovskiy) [inline tests/e2e/test_e2e_template.py]: if we have `train.py` in the code dir, let's also have a reasonable name in notebook?

File: {{cookiecutter.project_slug}}/Makefile (hunk 1 above)
Comment (mariyadavydova) [reply {{cookiecutter.project_slug}}/Makefile]: Yep, this makes sense to me.

File: tests/e2e/test_e2e_template.py (hunk 2 above)
Comment (mariyadavydova) [reply tests/e2e/test_e2e_template.py]: I'm all for a reasonable name. However, I think that `hello_world` is not really reasonable here. Let's rename it to `demo`.

The logic is the following: if you have `train.py`, you can simply add code there having a file with a meaningful name. If you have `hello_world.ipynb`, you will have to rename it in a real project, while `demo.ipynb` is a reasonable name for the notebook with model inference.

Comment (ayushkovskiy) [reply tests/e2e/test_e2e_template.py]: Totally agree, thank you!

File: {{cookiecutter.project_slug}}/Makefile (hunk 1 above)
Comment (ayushkovskiy) [reply {{cookiecutter.project_slug}}/Makefile]: https://github.com/neuromation/cookiecutter-neuro-project/issues/282

**Pull request merged**